knowledge-base/google-github-actions/auth/action-security.yml

```diff
@@ -0,0 +1,3 @@
+name: 'Authenticate to Google Cloud'
+# NOTE: GITHUB_TOKEN NOT USED
```

Comment:
If workload identity federation is used, the `id-token: 'write'` needs to be set. Check the examples in the readme file. This is similar to the existing aws authentication action - do check that action-security.yml. You will need to use conditional statements.